src/strategies/flashstake/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getAddress } from '@ethersproject/address';
+import { formatUnits } from '@ethersproject/units';
+import { multicall, subgraphRequest } from '../../utils';
+
+const FLASHSTAKE_SUBGRAPH_URL = {
+  '1':
+    'https://api.thegraph.com/subgraphs/name/blockzerohello/flash-stake-stats-v2-subgraph'
+};
+
+export const author = 'BlockzeroLabs';
+export const version = '0.1.0';
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const params = {
+    users: {
+      __args: {
+        where: {
+          id_in: addresses.map((address) => address.toLowerCase())
+        }
+      },
+      id: true,
+      liquidityPositions: {
+        liquidityTokenBalance: true,
+        user: {
+          id: true
+        },
+        pair: {
+          id: true,
+          token0: {
+            id: true
+          },
+          reserve0: true,
+          token1: {
+            id: true
+          },
+          reserve1: true,
+          totalSupply: true
+        }
+      }
+    }
+  };
+
+  const params2 = {
+    stakes: {
+      __args: {
+        where: {
+          user: addresses[0].toLowerCase()
```

Comment:
Here it seem that you are only query stake for a single address, the first address, you should instead query data for all the addresses.